index.d.ts

```diff
@@ -426,6 +426,8 @@ export interface Document {
   id?: string | number;
   _id: Types.ObjectId;
 
+  _doc?: Document;
```

Comment:
I think to be consistent with eg. https://github.com/guischdi/typed-mongoose/blob/66a1a2ee4f38095c721aff4efd819ee901815178/index.d.ts#L431 `Document` is alright